### PR TITLE
jreighley day 7 

### PR DESCRIPTION
**Pull request by jreighley**
corrected data file and unused exception warning.

### CODE REVIEW

File: src/aoc/y2018/d02/jreighley.cljc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                     (filter two?)
+                     (count)
+                     (pos?)))
+(def triple-count #(->> %
+                     (vals)
+                     (filter three?)
+                     (count)
+                     (pos?)))
+
+(def input-items
```
Comment (borkdude) [inline src/aoc/y2018/d02/jreighley.cljc]: Here you are doing work outside the tests. Can you wrap this in a delay or memoized fn? This way the test will time the performance of your solution correctly.

File: src/aoc/y2018/d07/jreighley.cljc [hunk 2]
```diff
@@ -0,0 +1,62 @@
+(ns aoc.y2018.d07.jreighley
+  (:require
+   [aoc.utils :as u :refer [deftest]]
+   [aoc.y2018.d07.data :refer [input answer-1 answer-2]]
+   [clojure.string  :refer [split-lines]]
+   [clojure.test :refer [is testing]]))
+
+(def sched
```
Comment (borkdude) [inline src/aoc/y2018/d07/jreighley.cljc]: Same here.

File: src/aoc/y2018/d07/jreighley.cljc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  (let [raw (split-lines input)
+        precedent-keys (sort-by first (map #(vector (subs % 5 6) (subs % 36 37 ))raw))]
+     precedent-keys))
+
+(def all-nodes (sort (reduce into #{} sched)))
```
Comment (borkdude) [inline src/aoc/y2018/d07/jreighley.cljc]: And here.

File: src/aoc/y2018/d07/jreighley.cljc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+(def remaining-nodes #(remove (set %) all-nodes))
```
Comment (borkdude) [inline src/aoc/y2018/d07/jreighley.cljc]: And here.

File: src/aoc/y2018/d02/jreighley.cljc (hunk 1 above)
Comment (jreighley) [reply src/aoc/y2018/d02/jreighley.cljc]: Moved Defs into memoized function

File: src/aoc/y2018/d07/jreighley.cljc (hunk 2 above)
Comment (jreighley) [reply src/aoc/y2018/d07/jreighley.cljc]: Moved Defs into memoized function

File: src/aoc/y2018/d07/jreighley.cljc (hunk 3 above)
Comment (jreighley) [reply src/aoc/y2018/d07/jreighley.cljc]: Moved Defs into memoized function

File: src/aoc/y2018/d07/jreighley.cljc (hunk 4 above)
Comment (jreighley) [reply src/aoc/y2018/d07/jreighley.cljc]: Moved Defs into memoized function

**Comment by borkdude** [comment]
@jreighley This branch has conflicts with src/aoc/y2018/d02/jreighley.cljc so I can't merge it.

One other thing to look at: https://github.com/borkdude/advent-of-cljc/blob/master/src/aoc/y2018/d07/data.cljc already exists. Why are you committing that file in this PR again?

File: src/aoc/y2018/d02/jreighley.cljc (hunk 1 above)
Comment (jreighley) [reply src/aoc/y2018/d02/jreighley.cljc]: This revision was requested on the d02 file -- that is the reason we have the conflict..

**Pull request merged**